Facade_Engine/Compute/AdjacentElements.cs

```diff
@@ -64,12 +64,13 @@ public static List<IElement2D> AdjacentElements(this IElement2D elem, List<IElem
             return adjElems;
         }
 
+
         /***************************************************/
 
-        [Description("Returns elements adjacent to a provided Element1D from a collection of elements")]
-        [Input("elem", "Element to find adjacencies at")]
-        [Input("refElems", "Elements to use to find element adjacencies")]
-        [Output("adjElems", "Adjacents elements to the provided element")]
+        [Description("Returns elements adjacent to a provided Element1D from a collection of elements.")]
+        [Input("elem", "Element to find adjacencies at.")]
+        [Input("refElems", "Elements to use to find element adjacencies.")]
+        [Output("adjElems", "Adjacents elements to the provided element.")]
```

Comment:
```suggestion
        [Input("element", "Element to find adjacencies at.")]
        [Input("referenceElements", "Elements to use to find element adjacencies.")]
        [Output("adjacentElements", "Adjacents elements to the provided element.")]
```

I don't see that we need to short-hand the descriptions/input variables really? Would be clearer to non-English-as-a-first-language coders if we didn't